release-notes.txt

```diff
@@ -38,43 +38,61 @@ New features and changes
 
 Community
 ---------
-    TBD
+    + Added GitHub feature request template
+    * Reworked CONTRIBUTING.md document
 
 Documentation
 -------------
     + New groups under `sys` (Serialization, Math, Memory Management,
       Hashes)
+    * Drivers with saul support are now also part of the drivers_saul group
 
 Core
 ----
     TBD
 
 System libraries
 ----------------
-    - lora_serialization: LoRa Serialization format support.
 
 Networking
 ----------
     TBD
 
 Packages
 --------
-    TBD
+    + lora_serialization: LoRa Serialization format support.
+    * semtech-loramac: bump to version 4.4.1
+    * semtech-loramac: use periph_eeprom to store lorawan configuration
+    * semtech-loramac: lorawan support is not considered experimental anymore
+
 Boards
 ------
-    TBD
+    + add support for samr30-xpro
+    + add i2c configuration for nucleo-f070
+    + add i2c configuration for nucleo-f722ze
+    + add second i2c configuration for b-l475e-iot01a
+
 CPU
 ---
-    TBD
+    + stm32l4: add support for periph_flashpage
+    * stm32-common: unified support for periph_i2c. I2C is now supported by all
+      stm32 families (F0/1/2/3/4/7, L0/1/4)
+
 Device Drivers
 --------------
-    TBD
+    + drivers/vcnl40x10: added initial support
+    * drivers/periph_i2c: rework API
+    * drivers/*: unify driver initialization parameters scheme
+    * drivers/ata8520e: rework internal implementation
+
 Build System / Tooling
 ----------------------
-    TBD
+    * pyterm: handle ctrl+d shortcut nicely
```

Comment:
this was not backported in the end :(. It changes an API

Reply:
This point is related to #8970, which was in master before the feature freeze, so there was no need for backporting.

Reply:
ah! sorry, I though it was #10105